packages/eslint-plugin-expo/tests/lib/rules/no-env-var-destructuring.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @fileoverview Disallow destructuring env vars from process.env
+ * @author Expo
+ */
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const RuleTester = require('eslint').RuleTester;
```

Comment:
Can we make these tests TypeScript and use ESM syntax?

Reply:
Pretty much all your comments are about stuff that [generator-eslint](https://github.com/eslint/generator-eslint) generated - I wasn't sold on it either, but I couldn't decide whether trade-off was worth all the boilerplate and docs getting created for you (so, easy to create new rules). But I guess this decides it 😄 and I'll rewrite this in TypeScript.